### PR TITLE
fix: add migration for mission places.json

### PR DESCRIPTION
**Pull request by braianj**
The previous migration `1669376178912_automatic-update-static-places.ts` imported an erroneous migration file. The file `08_places.json` was imported as it should have been `11_places.json`.

### CODE REVIEW

File: src/migrations/1669640960739_automatic-update-static-places.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { createPlaceMigration } from "../entities/Place/migration"
+import { PlaceAttributes } from "../entities/Place/types"
+import defaulPlace from "../seed/11_places.json"
```
Comment (fzavalia) [inline src/migrations/1669640960739_automatic-update-static-places.ts:3]:
```suggestion
import defaultPlace from "../seed/11_places.json"
```

File: src/migrations/1669640960739_automatic-update-static-places.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "like_rate",
+  "highlighted",
+  "featured",
+  "disabled",
+  "disabled_at",
+  "created_at",
+  "updated_at",
+]
+
+export const { up, down } = createPlaceMigration(defaulPlace, attributes)
```
Comment (fzavalia) [inline src/migrations/1669640960739_automatic-update-static-places.ts:31]:
```suggestion
export const { up, down } = createPlaceMigration(defaultPlace, attributes)
```

**Review by fzavalia — COMMENTED** [review]
There is no conflict between this migration and the previous "invalid" one right? They can both run and the migration state will be correct?

I have seen the migrations directory and there are lots of them for static places. I think there will be no issues. Fixing the typo is enough for the approve.

**Comment by braianj** [comment]
> There is no conflict between this migration and the previous "invalid" one right? They can both run and the migration state will be correct?
> 
> UPDATE: I have seen the migrations directory and there are lots of them for static places. I think there will be no issues. Fixing the typo is enough for the approve.

I deleted the Places that were erroneously added (to avoid duplication) and added them again.

**Pull request merged**